Formula/ripgrep.rb

```diff
@@ -15,11 +15,13 @@ class Ripgrep < Formula
   depends_on "asciidoc" => :build
   depends_on "docbook-xsl" => :build
   depends_on "rust" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pcre2" => :build
 
   def install
     ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
 
-    system "cargo", "install", "--root", prefix, "--path", "."
+    system "cargo", "install", "--root", prefix, "--path", ".", "--features", "pcre2"
```

Comment:
This will need to be knocked across two lines, since it has now passed 80 characters.

Reply:
Odd, I ran `brew audit --strict ripgrep` and rubocop didn't complain about this. And looking through audit.rb didn't bring up the 80 character limit, is that enforced or written down somewhere?

Reply:
It's enforced at a much higher limit right now, because otherwise we'd suddenly have, at best, hundreds of existing formulae being red-flagged by the audit. Eventually it'll work itself down to 80 characters, but in the mean time we do our best to stop people adding more lines we need to manually cut down later.

Reply:
Gotcha, I've made the change and moved them appropriately.

Reply:
Now the `--new-formula` flag makes sense for `brew audit`